docs/source/deployment/databricks/databricks_notebooks_development_workflow.md

```diff
@@ -0,0 +1,293 @@
+
+```ipython
+%pip install -r "/Workspace/Repos/<databricks_username>/iris-databricks/src/requirements.txt"
+```
+
+2. To run your project in your notebook, you must load the Kedro IPython extension. Add the following code to the second new cell to load the IPython extension:
+
+```ipython
+%load_ext kedro.ipython
+```
+
+3. Loading the extension allows you to use the `%reload_kedro` line magic to load your Kedro project. Add the following code to the third new cell to load your Kedro project:
+
+```ipython
+%reload_kedro /Workspace/Repos/<databricks_username>/iris-databricks
+```
+
+4. Loading your Kedro project with the `%reload_kedro` line magic will define four global variables in your notebook: `context`, `session`, `catalog` and `pipelines`. You will use the `session` variable to run your project. Add the following code to the fourth new cell to run your Kedro project:
+
+```ipython
+session.run()
+```
+
+After completing these steps, your notebook should match the following image:
+
+![Databricks completed notebook](../../meta/images/databricks_notebooks_workflow_finished_notebook.png)
+
+Run the completed notebook using the `Run All` button in the top right of the UI:
+
+![Databricks notebook run all](../../meta/images/databricks_run_all.png)
+
+On the first run of your Kedro project, you will be prompted to consent to analytics, type `y` or `N` in the field that appears and press `Enter`:
+
+![Databricks notebook telemetry consent](../../meta/images/databricks_telemetry_consent.png)
+
+You should see logging output while the cell is running. After execution finishes, you should see output similar to the following:
+
+```bash
+...
+2023-06-06 12:55:22,705 - iris_databricks.nodes - INFO - Model has an accuracy of 0.953 on test data.
+2023-06-06 12:55:22,709 - kedro.runner.sequential_runner - INFO - Completed 3 out of 3 tasks
+2023-06-06 12:55:22,709 - kedro.runner.sequential_runner - INFO - Pipeline execution completed successfully.
+```
+
+## Modify your project and test the changes
+
+Now that your project has run successfully once, you can make changes using the Databricks UI. In this section, you will modify the project to use a different ratio of training data to test data and check the effect of this change.
+
+### Modify the training / test split ratio
+
+The `databricks-iris` starter uses a default 80-20 ratio of training data to test data when training the classifier. You will edit this ratio to 70-30 and re-run your project to view the different result.
+
+In the Databricks workspace, click on the `Repos` tab in the side bar and navigate to `<databricks_username>/iris-databricks/conf/base/`. Open the the file `parameters.yml` by double-clicking it. This will take you to a built-in file editor. Edit the line `train_fraction: 0.8` to `train_fraction: 0.7`, your changes will automatically be saved.
```

Comment:
At the beginning I was not sure if this was supposed to commit the changes back to my GitHub private repository. Then by randomly clicking around I saw a "commit" interface (and saw the untracked `.telemetry` file from the consent above).

This guide is already a bit long so I'm not going to advocate for adding a paragraph on how to commit the changes, but just dropping the idea.